tutorials/advanced/elasticSnorkel.ipynb

```diff
@@ -0,0 +1,583 @@
+      "Index Information: \n",
+      " \n",
+      "health status index uuid pri rep docs.count docs.deleted store.size pri.store.size\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "eSearch.getIndices()\n",
+    "\n",
+    "#To delete an index by name or _all to delete all indices\n",
+    "# deleteIndex(\"indexName\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Indexing the Corpus\n",
+    "\n",
+    "Now that we have an established connection we can begin indexing. By default each document will contain a field corresponding to the sentence ID number, the sentence and an empty vector of 'o's (this will be useful for generating candidate tags below). Once everything has been indexed we can see its status with the amount of documents it contains as well as the size."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Beginning indexing\n",
+      "2591 items indexed\n",
+      "Index Information: \n",
+      " \n",
+      "health status index  uuid                   pri rep docs.count docs.deleted store.size pri.store.size\n",
+      "yellow open   corpus HCOENVAeRoKTrHduXdVChQ   5   1      67820            0       24mb           24mb\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "eSearch.elasticIndex(Document)\n",
+    "eSearch.getIndices()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Visualize Index\n",
+    "\n",
+    "To better visualize our data we get a mapping of index. We can see here that the each document in articles has 3 different fields: lineNum,candidates and sentence."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Index Mapping\n",
+      "{\n",
+      "  \"corpus\": {\n",
+      "    \"mappings\": {\n",
+      "      \"articles\": {\n",
+      "        \"properties\": {\n",
+      "          \"lineNum\": {\n",
+      "            \"type\": \"integer\"\n",
+      "          }, \n",
+      "          \"tagged\": {\n",
+      "            \"type\": \"text\", \n",
+      "            \"analyzer\": \"my_stop\"\n",
+      "          }, \n",
+      "          \"fillCand\": {\n",
+      "            \"type\": \"text\", \n",
+      "            \"analyzer\": \"my_stop\"\n",
+      "          }, \n",
+      "          \"sentence\": {\n",
+      "            \"type\": \"text\", \n",
+      "            \"analyzer\": \"my_analyzer\"\n",
+      "          }\n",
+      "        }\n",
+      "      }\n",
+      "    }\n",
+      "  }\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "eSearch.getIndexMap()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Visualizing a Document\n",
```

Comment:
Skip this too (or put this kind of stuff at the end)?  Again, goal should be to get a user up and using ES in a few lines!  Otherwise, realistically, no one will use...